lib/cocoapods/installer/analyzer/sandbox_analyzer.rb

```diff
@@ -230,7 +230,7 @@ def folder_exist?(pod)
         end
 
         def folder_empty?(pod)
-          Dir.glob(sandbox.pod_dir(pod) + '*').empty?
+          Dir.empty?(sandbox.pod_dir(pod))
```

Comment:
not a perf thing but this is newer API that we can now use because we dropped older rubies

Reply:
so I think this might technically be a change in behavior -- before, dotfiles would count. but i dont know if that's meaningful

Reply:
oh interesting - does `Dir.empty?` ignore dotfiles?

if so I'll revert this just in case

Reply:
given the one place it's used... i think it's fine?

Reply:
turns out this also fails if the folder itself doesn't exist. will revert

Reply:
alternatively, could checks !exists? || empty?

Reply:
the extra check will be a bit slow 😬 I think I'll revert for now as the change doesn't seem to be helpful